deploy/kubernetes/charts/helm-storage-backend/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
As those are Helm backends we will interact with K8s, so it will be also good to bootstrap cluster roles for them as they need to work across all Namespaces.

AFAIK we will need only:
- `get` and `list` on `configmaps`
- `get` and `list` on `secrets`

In case sth else will be needed, we can easily add it later if you will bootstrap them.